VagrantProvisionCentOS7.sh

```diff
@@ -56,7 +56,6 @@ sudo yum versionlock nodejs*
 
 # install useful and needed packages for working with hootenanny
 echo "### Installing dependencies from repos..."
-sudo localedef -i en_US -f UTF-8 en_US.UTF-8
```

Comment:
Do we know why this line is here?  It seems like I recall some core tests failing reading test files that maybe seem related to this.

Reply:
I think Nick added this line when he got Hoot working with Docker.

Reply:
Quoting my comment on the matter from DD-203:
> I believe the `localedef` command can be safely removed; it originates from a merge commit (099601e) and I think was used for containers or VMs that are using the `C` locale instead of `en_US.UTF-8` (PostgreSQL automatically takes system locale for the database). As our Vagrant boxes are already set to `en_US.UTF-8` the command is unnecessary.

Reply:
@NickolasTobin and I just worked through some stuff with this line to make it work with both vagrant and docker.  Best to check with him.

Reply:
I'm going to proceed as this is blocking progress on the CentOS 7 Jenkins pipeline, which won't be using Docker.  If you can identify the original reason, I'll take assignment of the issue.  I'll need details like the error it solved (why it was introduced) and the Docker image used.

Reply:
Removing or even moving that line breaks the docker image.

Reply:
@NickolasTobin that is why he has added the code in https://github.com/ngageoint/hootenanny/pull/1901

I'm assuming that everything is fixed now between the two.  You might want to pull down that branch and build a docker image to ensure that is correct.